### PR TITLE
Add e2e tests for create rke1 custom cluster

### LINKED ISSUE #8785: Automation: RKE1 provisioning member roles' "custom role" modal checkbox don't work correctly
Based on the work done in https://github.com/rancher/dashboard/issues/8621, let's write an automated test to verify that when we create or edit an RKE1 cluster and the membership section allows users to define custom roles, clicking the label of each checkbox actually selects it. 

Previous, selections are only saved when clicking the box itself.

@Shavindra may be able to help here.

To reproduce:

- Navigate to Cluster Management > Cluster > Edit Cluster > Cluster Roles
- Select user and Custom role
- Click on the labels of the roles. It should select the different roles
- When click on save it should create all the roles
- This can be verified by navigating to Cluster Explorer > Cluster > Cluster & Project members and sorting the list by age to view recently created roles.

### PR DESCRIPTION
**Pull request by richard-cox**
### Summary
Fixes #8785

### Occurred changes and/or fixed issues
- Depends on https://github.com/rancher/ui/pull/5035. This contains functional changes that shouldn't merge until 2.7q2 is out. 

### Technical notes summary
- also updated lots of ~/ @/

### CODE REVIEW

File: cypress/e2e/po/components/ember/ember-accordion.po.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    super(`.accordion:has(> [data-testid="${ rootTestId }__header"])`); // This should work, but given current functionality it isn't used
+    this.headerSelector = `[data-testid="${ this.rootTestId }__header"]`;
+    this.contentSelector = `[data-testid="${ this.rootTestId }__content"]`;
+  }
+
+  expand() {
+    this.header().click();
+  }
+
+  formMembers() {
```
Comment (mantis-toboggan-md) [inline cypress/e2e/po/components/ember/ember-accordion.po.ts:18]: Not sure about this, wouldn't we end up adding properties here for every use of the accordion we want to test? Feels kinda backwards like the accordion should be a property on the formMembers class

File: cypress/e2e/po/components/ember/ember-component.po.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import ComponentPo from '@/cypress/e2e/po/components/component.po';
+
+export default class EmberComponentPo extends ComponentPo {
+  // Note - only selector ctor allowed
```
Comment (mantis-toboggan-md) [inline cypress/e2e/po/components/ember/ember-component.po.ts:4]: looks like a typo in the comment here

File: cypress/support/commands.ts [hunk 3]
```diff
@@ -109,3 +109,14 @@ Cypress.Commands.add('requestBase64Image', (url: string) => {
       return Cypress.Blob.blobToBase64String(blob);
     });
 });
+
+Cypress.Commands.add('iFrame', () => {
+  // This creates a lot of entries in the cypress running bar...
+  // in theory adding `{ log: false }` should hide these ...
+  // it does not
+  return cy
```
Comment (mantis-toboggan-md) [inline cypress/support/commands.ts:117]: You could reduce the number of log entries by adding `{ log: false }` to the `get` and `its` calls but not `should` [(yet?)](https://github.com/cypress-io/cypress/issues/7693) . Idk if that's really any clearer though; you just lose context for the `should`

**Review by mantis-toboggan-md — CHANGES_REQUESTED** [review]

File: cypress/support/commands.ts (hunk 3 above)
Comment (richard-cox) [reply cypress/support/commands.ts:120]: i think that's a reasonable change, given how these obfuscate all the other content

**Comment by mantis-toboggan-md** [comment]
The code looks fine aside from linting errors, but I think e2e ci tests wont pass until a new v2.7-head rancher/rancher image is available. Since we're running our e2e rancher docker instance with `-e CATTLE_UI_OFFLINE_PREFERRED=true` we need the bundled ember code to include your changes [here](https://github.com/rancher/ui/pull/5035) right? That was merged 3 days ago but the [latest v2.7-head rancher/rancher image](https://hub.docker.com/r/rancher/rancher/tags?page=1&name=v2.7-head) is 9 days old.

**Comment by richard-cox** [comment]
Hadn't managed to get around to debugging this, good find! We need to bring in the latest rancher/ui build and serve it up as per the new way we build and server the dashboard. I've created https://github.com/rancher/dashboard/issues/9217 to cover this. Otherwise the last RC build will be used

**Review by mantis-toboggan-md — APPROVED** [review]

**Pull request merged**